lib/manager/gradle/__snapshots__/index.spec.ts.snap

```diff
@@ -610,7 +610,7 @@ Array [
     },
   },
   Object {
-    "cmd": "docker run --rm --name=renovate_gradle --label=renovate_child -v \\"localDir\\":\\"localDir\\" -w \\"localDir\\" renovate/gradle bash -l -c \\"./gradlew --init-script renovate-plugin.gradle renovate\\"",
+    "cmd": "docker run --rm --name=renovate_gradle --label=renovate_child -v \\"localDir\\":\\"localDir\\" -w \\"localDir\\" renovate/gradle bash -l -c \\"gradle --init-script renovate-plugin.gradle renovate\\"",
```

Comment:
The command changes from `./gradlew` to `gradle` here because this was previously wrong: We have another test case for the `./gradlew` case, in this instance, it should be `gradle`.